css/css-basics_pokemon-page/css/styles.css

```diff
@@ -45,7 +51,15 @@
 */
 
 section {
-  /* your solution */
+  background-color: white;
+  margin: 10px;
+  padding: 20px;
+  border-style: solid;
```

Comment:
I think this puts a border on all all-around, but the task said _Add a solid 5px border with the color "orange" that is **only visible on the left. Hint: Use "border-left"**. :)_